core/fetcher_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func TestBlockFetcher_GetBlock_and_SubscribeNewBlockEvent(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
```

Comment:
I was probably a bit too conservative with increasing these times, but they must be increased to keep tests passing

openned https://github.com/celestiaorg/celestia-app/issues/2199 to figure out why